src/js/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -27,6 +28,10 @@ const RadioButtonGroup = forwardRef(
     ref,
   ) => {
     const formContext = useContext(FormContext);
+    const theme = useContext(ThemeContext) || defaultProps.theme;
+    const gap = theme.global.radioButtonGroup
+      ? theme.global.radioButtonGroup.gap
+      : 'small';
```

Comment:
Would setting a default value here will be backward compatible?

Reply:
in the original code radiobuttongroup had a default value of small.
```
const RadioButtonGroup = forwardRef(
  (
    {
      children,
      disabled,
      gap = 'small',
      name,
      onChange,
      options: optionsProp,
      value: valueProp,
      ...rest
    },
    ref,
  ) => {
```

Reply:
We should keep the behavior of the original code and add the theme option for the `gap` without setting a default value on it. That will be more backward compatible in case a user is overriding the theme.

Reply:
changes made